include/exadg/postprocessor/kinetic_energy_spectrum.cpp

```diff
@@ -250,31 +255,20 @@ class DealSpectrumWrapper
   }
 
 private:
-  types::global_dof_index
-  pow_(const types::global_dof_index base, const types::global_dof_index exp)
-  {
-    types::global_dof_index result = 1.0;
-
-    for(types::global_dof_index i = 0; i < exp; i++)
-      result *= base;
-
-    return result;
-  };
-
```

Comment:
This is no longer used?

Reply:
Indeed, this is the same as `Utilities::pow()` - I made sure that the argument in `pow` is `types::global_dof_index` to also work with large sizes.